src/components/MonitoringPlanEvaluationReport/MonitoringPlanEvaluationReport.js

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useRef } from "react";
+import { Button } from "@trussworks/react-uswds";
+import { DataTableRender } from "../DataTableRender/DataTableRender";
+
+
+export const MonitoringPlanEvaluationReport = ({
+}) => {
+
+    //Columns to be displayed in the data table
+    const columnNames = [
+        "Unit/Stack",
+        "Severity",
+        "Category",
+        "Check Code",
+        "Result Message"
+    ];
+
+    //LOAD FROM API CALL
+    const data = [{
+        col1: "Sample U/S 1",
+        col2: "Severity 1",
+        col3: "category 1",
+        col4: <Button
+            type="button"
+            unstyled={true}
+            className="text-primary text-underline"
+            href="#"
+            role="link"
+            rel="Check Code 1"
+        // title={`Go to ${ele.name} page`}
```

Comment:
please remove commented out content if not using

Reply:
Removed